packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -72,6 +69,10 @@ const HomepageListProvider = () => {
     );
   }
 
+  const query = gqlCountQueries(lists);
```

Comment:
@Vultraz is right, you can not move the hooks after a if block returning early.

Reply:
Sorry. Maybe I am out in too deep water here. I made a new commit with a conditional query and added skip to useQuery. It looks a little dirty, but it was the best I could come up with.

Reply:
I think you could use `useLazyQuery` instead and simply invoke it after the no-list bailout

Reply:
Ahh, nice. That's it. useLazyQuery feels much better.